libr/core/cmd_print.c

```diff
@@ -4847,7 +4848,7 @@ static int cmd_print(void *data, const char *input) {
 				char *s = malloc (core->blocksize + 1);
 				int i, j;
 				if (s) {
-					memset (s, 0, core->blocksize);
+					memset (s, 0, core->blocksize + 1);
```

Comment:
oh wait, remove the memset keep the malloc, and null terminate the string at the end of the for loop